packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -11,25 +9,10 @@ export const shouldMatchSnapshot = (title, component) => {
   });
 };
 
-export const shallowRender = component => {
-  const shallowRenderer = new ShallowRenderer();
-  shallowRenderer.render(component);
-  const result = shallowRenderer.getRenderOutput();
-
-  return result;
-};
-
-export const shouldShallowMatchSnapshot = (title, component) => {
-  it(title, () => {
-    const tree = shallowRender(component);
-    expect(tree).toMatchSnapshot();
-  });
-};
-
 export const isNull = (title, component) => {
   it(title, () => {
-    const tree = renderer.create(component).toJSON();
-    expect(tree).toBeNull();
+    const { container } = render(component);
+    expect(container.firstChild).toBeNull();
```

Comment:
Is this the same thing? what is first child always the top level thing? could you have a second child that was not null?

Reply:
How would that look like? First child null and second not null. AFAIK `null` is not rendered which makes the second child that was not null to be the firstChild.

Reply:
Maybe we could check if the container has children and assert that their are no children rendered.